public/app/features/dimensions/utils.ts

```diff
@@ -107,7 +107,7 @@ export function findField(frame?: DataFrame, name?: string): Field | undefined {
   return undefined;
 }
 
-export function findFieldIndex(frame?: DataFrame, name?: string): number | undefined {
+export function findFieldIndex(frame?: DataFrame, name?: string, frames?: DataFrame[]): number | undefined {
```

Comment:
@ryantxu can we just make this signature similar to the internal `getFieldDisplayName(name, frame, frames)`?

Reply:
yup -- internal so yes